### ISSUE TITLE
Remote debugging support

### ISSUE THREAD

**Issue by davidwengier** [dotnet/project-system#5767]
Following on from #3756 we need to enable support for all of the features of remote debugging that the current ProjectFlavoring system supports like port providers etc.

**Comment by drewnoakes** [comment]
https://github.com/dotnet/project-system/pull/5831 brings in some bits that'll be useful for supporting authentication mode, if that's part of the solution here.

**Comment by davidwengier** [comment]
#5860 will add authentication port providers etc. This issue will be mainly around Deploy

**Comment by davkean** [comment]
We've put this on hold for the moment.

**Comment by MagicAndre1981** [comment]
> We've put this on hold for the moment.

why? Remote debug Winforms applications was always a pain and I would like to have the good UWP remote debug experience

**Comment by davkean** [comment]
We've put the deployment side of remote debugging on hold (deploy the application on F5 to a remote machine) while we figure out where UWP/WinUI fits into .NET 5. Despite that, we have improved remote debugging in 16.5 (where we first added support) and 16.6 (added support for selecting authentication and UI for selecting remote machines).

**Comment by MagicAndre1981** [comment]
> We've put the deployment side of remote debugging on hold (deploy the application on F5 to a remote machine) while we figure out where UWP/WinUI fits into .NET 5. 

ok,, thanks for this information, but I see no correlation to UWP. I just want to press F5 and the app gets automatically deployed to remote target without using PostBuildEvent to copy the file on my own. This was always horrible.

**Comment by davkean** [comment]
Yes and this issue is tracking it. It's currently on the backlog and has has not yet been scheduled.

**Comment by MagicAndre1981** [comment]
any update? With [.NET 5 UWP is moved into its own package](https://github.com/microsoft/CsWinRT), the future of UWP is clear you could go forward and give us updates what you plan about deployment after build.

**Comment by MagicAndre1981** [comment]
any update on this? Will this be solved in VS2022 or still in 2019 16.11?
